django_tasks/backends/database/backend.py

```diff
@@ -88,3 +91,18 @@ async def aget_result(self, result_id: str) -> TaskResult:
             return (await DBTaskResult.objects.aget(id=result_id)).task_result
         except (DBTaskResult.DoesNotExist, ValidationError) as e:
             raise ResultDoesNotExist(result_id) from e
+
+    def check(self, **kwargs: Any) -> List[CheckMessage]:
+        if "django_tasks.backends.database" not in settings.INSTALLED_APPS:
+            backend_name = self.__class__.__name__
+            package_name = sys.modules[__name__].__package__
```

Comment:
Given we're hardcoding the module name above, we should just hardcode it here too.